src/style.css

```diff
@@ -2,2 +1,71 @@
+* {
   margin: 0;
+  padding: 0;
+}
+
+.html {
+  --header-link-color: #00acdc;
+}
+
+.body {
+  margin: 0;
+}
```

Comment:
Similar to the `.html` class, it is unusual to have a `.body` class. If you are trying to style the `<body>` element, you should use the `body` tag selector.